keel-core/src/main/kotlin/com/netflix/spinnaker/keel/verification/VerificationRunner.kt

```diff
@@ -67,7 +89,7 @@ class VerificationRunner(
   }
 
   private val Collection<Pair<*, ConstraintStatus?>>.anyStillRunning: Boolean
-    get() = any { (_, status) -> status == PENDING }
+    get() = !none { (_, status) -> status == PENDING }
```

Comment:
Fun fact: `any` returns `false` for an empty collection which is not what we want here. "Is anything still running?" should be `true` if there are no things at all.

Reply:
LOL well, who knew.

Reply:
What's the reasoning behind treating empty collection as true instead of false? When would we even call this on an empty collection? Looking at the code, it doesn't seem like should ever happen since we return early if there are no verifications defined on an environment..